datalad/ssh_repo_elm.md

```diff
@@ -0,0 +1,98 @@
+## 🧠 Datalad for Students: Minimal Reproducible Workflow
+
+### 📦 1. Create a Datalad Dataset for Data (on `elm`)
+
+**On your local machine:**
+
+```bash
+datalad create -c text2git image10k-zooniverse
+cd image10k-zooniverse
+```
+
+**Annex the big files (e.g. CSVs):**
+
+```bash
+echo "*.csv annex.largefiles=anything" >> .gitattributes
+datalad save -m "Set annex rules for CSVs"
+```
+
+**Push data to `elm`:**
+
+```bash
+datalad create-sibling \
+  --name elm \
+  --site datalad \
+  --sshurl ssh://elm/data/simexp/pbellec/image10k-zooniverse \
```

Comment:
`--sshurl` does not exists, it is a positional arg.

Reply:
see comment above